atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1203,8 +1206,9 @@ public void commitThrowsIfRolledBackAtCommitTime_expiredLocks() {
         PreCommitCondition condition =
                 unused -> doReturn(ImmutableSet.of()).when(timelockService).refreshLockLeases(any());
 
-        LockImmutableTimestampResponse res = timelockService.lockImmutableTimestamp();
-        long transactionTs = timelockService.getFreshTimestamp();
+        ConjureStartTransactionsResponse conjureResponse = startTransactionWithWatches();
+        LockImmutableTimestampResponse res = conjureResponse.getImmutableTimestamp();
+        long transactionTs = conjureResponse.getTimestamps().start();
```

Comment:
probably worth extracting a common method out for this

Reply:
sadly there are two outputs (res and transactionTs); I did the best I could by extracting out `startTransactionWithWatches()`.